apis/v1alpha1/gateway_types.go

```diff
@@ -770,27 +741,28 @@ const (
 	// interoperability.
 	ListenerConditionResolvedRefs ListenerConditionType = "ResolvedRefs"
 
-	// ListenerReasonDegradedRoutes indicates that not all of the routes
-	// selected by this Listener could be configured. The specific reason
-	// for the degraded route should be indicated in the route's
-	// .Status.Conditions field.
+	// This reason is used with the "ResolvedRefs" condition
```

Comment:
Is there any way we can make this linking more obvious/automatic/structured in the reference docs?

Reply:
We can manually link it. Could try using separate type aliases for the reasons for each condition type, but might be awkward/overkill.

Reply:
We din't like manual linking and there's no reasonable way to automatically link AFAICT.